### DISCUSSION TITLE
Questions about normal and tangential forces during a collisons

### DISCUSSION THREAD

**Discussion by youyunxu** [chaos-polymtl/lethe#1002] [Q&A]
Hi, 

I'm new to lethe. I want to know how to get the figure 10 and 11 in "Lethe-DEM : An open-source parallel discrete element
solver with load balancing". When I tried to use nonlinear viscoelastic model to calculate by hand, it seems different with the figures.
Here are the figures in document:
![Screenshot 2024-01-28 133728](https://github.com/lethe-cfd/lethe/assets/31919481/99bd1064-9d7b-4fb3-afc9-5eefe2c8de8d)

Many thanks!
Xu

**Comment by blaisb** [comment]
Dear @youyunxu I am unsure what you mean by get them by hand?
The viscoelastic model used in Lethe are all fully described in the Theory guide;
https://lethe-cfd.github.io/lethe/documentation/theory/dem/dem.html

Those are the formulas that are currently used in the software. It is possible that very small modifications were made to the contact model between the Lethe-DEM paper and the current implementation. Our current implementation is tested on multiple benchmark including normal contact and contacts at an angle. See the following two examples:
https://lethe-cfd.github.io/lethe/documentation/examples/dem/bouncing-particle/bouncing-particle.html
https://lethe-cfd.github.io/lethe/documentation/examples/dem/oblique-wall-impact/oblique-wall-impact.html

Don't hesitate if you have any more questions
Best
Bruno

**Reply by youyunxu** [reply]
Dear Bruno,

Thanks a lot. I just plug parameters in the paper to the viscoelastic model formulas and get a big normal force (I specified a normal overlap equals 0.05 and normal relative velocity equals 0.1). I think relative values are used in the figure. I will try to use more examples.

Best
xu

**Reply by blaisb** [reply]
Ohh yes, here delta_n* is the normalized overlap compared to the maximal overlap reached. So 0.05 is a huuuuge value for an overlap... In the figure above, when you have 0.05 it's actually 0.05 * the maximum overlap reached during contact. (so depending on the Young modulus it can be as small as 0.0001 d_p for example)
